src/validators.ts

```diff
@@ -14,7 +14,7 @@ export const validatePath = (input: string): string => {
   if (input.match(/^(\/|https?:\/\/)/)) {
     return input;
   } else {
-    throw `Bad path "${input}". Skipping.`;
+    throw new Error(`Bad path "${input}". Skipping.`);
```

Comment:
Best practice (and better for testing too) to throw an actual Error object instead of a string.